### PR TITLE
add(tutorials): exporting yolo world model

### PR DESCRIPTION
**Pull request by legendof-selda**
This allows us to export yolo world onnx model which can be later used in mobile inference.

### CODE REVIEW

**Comment by legendof-selda** [comment]
@microsoft-github-policy-service agree

File: tutorials/yolov8_world_e2e.py [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+        By default the post processing will scale the bounding boxes and key points to the original image.
+    """
+    )
+    parser.add_argument("model", type=Path, help="The ONNX YOLOv8 World model.")
+    parser.add_argument(
+        "--classes",
+        type=Path,
+        default="classes.json",
+        help="JSON file containing list of classes that will be set as yolo world prompt.",
+    )
```
Comment (skottmckay) [inline tutorials/yolov8_world_e2e.py:313]: Where should a user get this file from in order to run the script?

Comment (legendof-selda) [reply tutorials/yolov8_world_e2e.py:313]: In yolo world you specify prompts to the model. So, it's just a list of text/classes that you will prompt the model for zero-shot detection. So, it's user specified. The user has to create the list of prompts.

Comment (skottmckay) [reply tutorials/yolov8_world_e2e.py:313]: Ideally a tutorial is self-contained with default inputs that allow running it end-to-end. We want it to make it as simple as possible for the user to see it working.

Is there an example classes.json that could be added that works with the default `--test_image` value of data/stormtroopers.jpg?

Comment (legendof-selda) [reply tutorials/yolov8_world_e2e.py:313]: I can add something. Should I create a file and commit it or have user pass in a list of values as classes and we can set default example classes for the argument?

Comment (legendof-selda) [reply tutorials/yolov8_world_e2e.py]: I have provided default values for classes as requested. Please review @skottmckay

**Review by natke — APPROVED** [review]

**Pull request merged**